packages/components/src/font-size-picker/test/index.js

```diff
@@ -220,9 +220,16 @@ describe( 'FontSizePicker', () => {
 						value={ fontSizes[ 0 ].size }
 					/>
 				);
-				const element = screen.getByLabelText( 'Large' );
-				expect( element ).toBeInTheDocument();
-				expect( element.children[ 0 ].textContent ).toBe( '3' );
+				const largeElement = screen.getByLabelText( 'Large' );
+				expect( largeElement ).toBeInTheDocument();
+				expect( largeElement.children[ 0 ].textContent ).toBe( 'L' );
```

Comment:
```suggestion
				expect( largeElement ).toHaveTextContent( 'L' );
```

Can we leverage `toHaveTextContent()` here? Appears to work for me and might help improve readability of quite a few tests.